lib/shopify_api/only_send_mutated_params.rb

```diff
@@ -0,0 +1,34 @@
+module ShopifyAPI
+  module OnlySendMutatedParams
```

Comment:
Hmm, I would like to differentiate between this and something that gives you access to  dirty attributes like `ActiveModel::Dirty`, how about something like `OnlySyncDirtyAttributes` which removes the language thats http request specific?